contents/java/JUnit5.md

```diff
@@ -0,0 +1,259 @@
+<frontmatter>
+  title: Introduction to JUnit 5
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Testing in Java with JUnit 5
+
+Author: [Lin Si Jie](https://github.com/sijie123)
+
+Reviewers: [Aadyaa Maddi](https://github.com/amad-person), [Marvin Chin](https://github.com/marvinchin)
+
+## What is JUnit 5?
+JUnit is an open-source automated testing framework for Java. Using its vast API library, you can easily write and run tests to ensure correctness in your software engineering project. JUnit 5 is the latest iteration of the JUnit project, supporting multiple new features in Java (such as Lambdas) and adopting a new <popover effect="fade" content="think: OOP for testing" placement="top">Extension-based model</popover> to make writing tests easier, cleaner, and more modular.
+
+## Why do automated testing?
+There are many reasons why automated testing is important for any software engineering project.
```

Comment:
Modal should be linked to the first appearance of the keyword?.
![image](https://user-images.githubusercontent.com/1673303/55548082-4a07f800-5705-11e9-8c3e-18d0a044bf31.png)